### ISSUE TITLE
Reproduce results on MNLI dataset

### ISSUE THREAD

**Issue by ofirzaf** [benzakenelad/BitFit#3]
Hi, 

I had trouble reproducing the results you report in the paper for MNLI. I am using the default example you have in the README and the learning rate you mention in the paper for BERT-Base.

```bash
python run_glue.py \
        --output-path $1 \
        --task-name mnli\
        --model-name bert-base-cased\
        --fine-tune-type bitfit\
        --learning-rate 1e-4\
        --gpu-device 0
```

Anything I need to change/doing wrong?

Thanks

**Comment by benzakenelad** [comment]
Hey Ofir,

For how many epochs did you train?

**Comment by ofirzaf** [comment]
epochs 16, batch size 8. Used the default values of the script.

**Comment by benzakenelad** [comment]
Can you please share the learning curves plot?

**Comment by benzakenelad** [comment]
(It’s in the output dir)

**Comment by ofirzaf** [comment]
Here you go
![learning_curves_accuracy](https://user-images.githubusercontent.com/18296312/199100054-88d9c579-9f30-4b19-8927-637eb7205fda.png)

**Comment by benzakenelad** [comment]
It seems learning curves still increase, try to increase the number of
epochs (try 32)

**Comment by benzakenelad** [comment]
Please lmk if that works

**Comment by ofirzaf** [comment]
Looks like it pretty much converges on these results. AFAIK, full fine-tuning of BERT reaches about 3 points higher.
Do you have an idea what is not working? Maybe something in the 
![learning_curves_accuracy](https://user-images.githubusercontent.com/18296312/199605156-806c812f-e69c-47c6-94ca-818fc4cd0dea.png)

**Comment by benzakenelad** [comment]
Try using learning rate of 4e-4 (it seems it still goes up..). Also, it
doesn’t suppose to reach full fine-tuning performance for MNLI (see Table 3
to in the paper)

**Comment by ofirzaf** [comment]
I will try that, thanks a lot for your help!
